pep-0503.txt

```diff
@@ -54,12 +54,12 @@ a repository. The format of this URL is ``/<project>/`` where the ``<project>``
 is replaced by the normalized name for that project, so a project named
 "HolyGrail" would have an URL like ``/holygrail/``. This URL must respond with
 a valid HTML5 page with a single anchor element per file for the project. The
-text of the anchor tag **MUST** be the filename of the file and the href
-attribute **MUST** be an URL that links to the location of the file for
-download. The URL **SHOULD** include a hash in the form of an URL fragment with
-the following syntax: ``#<hashname>=<hashvalue>``, where ``<hashname>`` is the
-lowercase name of the hash function (such as ``sha256``) and ``<hashvalue>`` is
-the hex encoded digest.
+href attribute **MUST** be an URL that links to the location of the file for
```

Comment:
```suggestion
href attribute **MUST** be a URL that links to the location of the file for
```

There's a convention for initialisms that uses the way they're said to decide between "a" and "an", rather than the way they're written. So in this case, "URL" is usually said as "you-are-ell", so it becomes "a URL" rather than "an URL". The latter isn't technically *wrong*, it just looks a bit odd (and doesn't match the way you'd read the sentence out loud).

Reply:
There’s another occurrence of “an URL” above, from the original document. (There might be mor, I didn’t check.) Should I also change it while I’m on this?

Reply:
Aye, may as well.